_transactions.md

```diff
@@ -769,3 +769,136 @@ Returns a [Transaction Object](#transaction-object).
 <aside class="notice">
   Note that you will only receive the list of committed transactions.
 </aside>
+
+## Transaction Limit Errors
+
+> Example of a transaction that fails due to for insufficient funds in the origin card:
+
+```bash
+curl 'https://api.uphold.com/v0/me/cards/a6d35fcd-xxxx-9c9d1dda6d57/transactions?commit=true' \
+  -H "Authorization: Bearer <token>" \
+  -H "Content-Type: application/json" \
+  -d '{ "denomination": { "amount": "1000", "currency": "EUR" }, "destination": "bc9b3911-4bc1-4c6d-ac05-0ae87dcfc9b3" }'
+```
+
+```json
+{
+  "code": "validation_failed",
+  "errors": {
+    "denomination": {
+      "code": "validation_failed",
+      "errors": {
+        "amount": [
+          {
+            "code": "sufficient_funds",
+            "message": "Not enough funds for the specified amount"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+> Example of a transaction that hits the maximum amount for the denomination currency:
+
+```bash
+curl 'https://api.uphold.com/v0/me/cards/a6d35fcd-xxxx-9c9d1dda6d57/transactions?commit=true' \
+  -H "Authorization: Bearer <token>" \
+  -H "Content-Type: application/json" \
+  -d '{ "denomination": { "amount": "1000", "currency": "BTC" }, "destination": "bc9b3911-4bc1-4c6d-ac05-0ae87dcfc9b3" }'
+```
+
+```json
+{
+  "code": "validation_failed",
+  "errors": {
+    "denomination": {
```

Comment:
In most cases, this error will appear, but there are some cases where it doesn't, and only `destination` or `origin` will be shown.

Reply:
I've simplified the examples and truncated the returned JSON, both for readability/brevity, and to make them more about illustrating the error structure than exactly replicating specific error cases. As part of the truncation I removed the `denomination` part of the error.